components/slider/index.css

```diff
@@ -207,6 +207,9 @@
 	padding-inline-end: var(--mod-slider-handle-gap, var(--spectrum-slider-handle-gap));
 	margin-inline-start: var(--mod-slider-track-margin-offset, var(--spectrum-slider-track-margin-offset));
 
+	touch-action: none;
+	user-select: none;
```

Comment:
You left really great comments in color handle CSS regarding these two properties. Did you want to add them here as well? 

Not a blocker if you don't feel like it- I just really liked the comments! 😆

Reply:
@aramos-adobe I was trying to test this, and struggling with the slider. I wonder if it's because `.spectrum-Slider-track` and `.spectrum-Slider-fill` are only like, 2px high. This is the only component I feel like I can't really verify.

I also ended up with another question -do you think touch-action should go on the slider handle, also? What about the `.spectrum-Slider-track ~ .spectrum-Slider-track` selector?